docs/OverseeingIssuePrLifeCycle.md

```diff
@@ -18,18 +18,21 @@ When an issue comes in,
 
 * A PR should have,
   * **1 or more `reviewers`** (See [Reviewing PRs](ReviewingPrs.md) for more info)
-  * **no more than one `assignee`**: The `assignee` field can be used to indicate the main reviewer when there are 
-    multiple reviewers and it is not clear who is the main reviewer. Otherwise it can be left empty.
+  * **all reviewers should be selected as `assignees`**
+  
+  > Duplicating `reviewers` as `assignees` is needed because GitHub filter syntax cannot filter by reviewers yet.
```

Comment:
@acjh Is the feature working for you? In my case, it shows 0 open and 0 closed review requests.